ome_zarr/writer.py

```diff
@@ -432,8 +435,85 @@ def write_image(
         axes=axes,
         coordinate_transformations=coordinate_transformations,
         storage_options=storage_options,
+        **metadata,
+    )
+
+
+def write_multiscale_image_labels(
+    pyramid: List,
+    group: zarr.Group,
+    name: str,
+    chunks: Union[Tuple[Any, ...], int] = None,
+    fmt: Format = CurrentFormat(),
+    axes: Union[str, List[str], List[Dict[str, str]]] = None,
+    coordinate_transformations: List[List[Dict[str, Any]]] = None,
+    colors: List[JSONDict] = None,
+    properties: List[JSONDict] = None,
+    storage_options: Union[JSONDict, List[JSONDict]] = None,
+    **metadata: JSONDict,
+) -> None:
+    """
+    Write image labels in pyramidal format with multiscale and image-label metadata to disk.
+
+    Creates the label data in the sub-group "labels/{name}"
+
+    pyramid: List of np.ndarray
+      the image label data to save. Largest level first
+      All image arrays MUST be up to 5-dimensional with dimensions
+      ordered (t, c, z, y, x)
+    group: zarr.Group
+      the group within the zarr store to store the data in
+    name: str
+      the name of this labale data
+    chunks: int or tuple of ints,
+      size of the saved chunks to store the image
+    fmt: Format
+      The format of the ome_zarr data which should be used.
+      Defaults to the most current.
+    axes: str or list of str or list of dict
+      List of axes dicts, or names. Not needed for v0.1 or v0.2
+      or if 2D. Otherwise this must be provided
+    coordinate_transformations: 2Dlist of dict
+      For each path, we have a List of transformation Dicts.
+      Each list of dicts are added to each datasets in order
+      and must include a 'scale' transform.
+    colors: list of dict
```

Comment:
The primary downside of these explicit declaration is that it prevents someone to use this wrapper to store additional `image-label` metadata.
Another option would be to have pass the entire dictionary for `image-label` as a single argument?

Reply:
You're right, it's probably better to remove `colors` and `properties` in favor of a dict argument `image_label_metadata` (or similar name).